vllm/model_executor/layers/rejection_sampler.py

```diff
@@ -135,6 +149,35 @@ def _batch_modified_rejection_sampling(
 
         return accepted, recovered_token_ids
 
+    def _create_uniform_samples(self,
+                                seeded_seqs: Optional[Dict[int,
+                                                           torch.Generator]],
+                                batch_size: int, k: int,
+                                device: torch.device) -> torch.Tensor:
+        if not seeded_seqs:
+            uniform_rand = torch.rand(batch_size, k + 1, device=device)
```

Comment:
nit: suggest returning immediately, so readers do not have to consider this branch when reading the rest of the function